src/content/en/updates/2018/01/paintapi.md

```diff
@@ -0,0 +1,278 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Houdini’s CSS Paint API allows you to programmatically draw CSS image resources.
+
+{# wf_updated_on: 2018-01-18 #}
+{# wf_published_on: 2018-01-18 #}
+{# wf_tags: css,style,houdini,javascript #}
+{# wf_featured_image: /web/updates/images/2018/01/paintapi/houdinidiamond.png #}
+{# wf_featured_snippet: Houdini’s CSS Paint API allows you to programmatically draw CSS image resources. #}
+{# wf_blink_components: Blink>CSS #}
+
+
+# CSS Paint API {: .page-title }
+
+{% include "web/_shared/contributors/surma.html" %}
+
+## New possibilities in Chrome 65
+CSS Paint API (also known as “CSS Custom Paint” or “Houdini’s Paint Worklet”) is
+about to be enabled by default in Chrome Stable. What is it? What can you do
+with it? And how does it work? Well, read on, will ya’…
+
+
+CSS Paint API allows you to programmatically generate an image whenever a CSS
+property expects an image. Properties like `background-image` or `border-image`
+are usually used with `url()` to load an image file or with CSS built-in
+functions like `linear-gradient()`. Instead of using those, you can now use
+`paint(myPainter)` to reference a _paint worklet_.
+
+### Writing a paint worklet
+
+To define a paint worklet called `myPainter`, we need to load a CSS paint
+worklet file using `CSS.paintWorklet.addModule('my-paint-worklet.js')`. In that
+file we can use the `registerPaint` function to register a paint worklet class:
+
+```js
+class MyPainter {
```

Comment:
Im not sure you need this. Your checkboard example is complete and minimal. Maybe just build on that?